### ISSUE TITLE
Does not work, no rights even with full disk access

### ISSUE THREAD

**Issue by runxel** [gingerbeardman/stapler#8]
Gave Stapler full disk access, I still can not get it to work.
Instead it always complains about not having any rights:
![grafik](https://github.com/user-attachments/assets/bb4cad87-8947-4a65-aa68-50be46017b06)

**Comment by gingerbeardman** [comment]
Please try the troubleshooting steps in this comment: https://github.com/gingerbeardman/stapler/issues/6#issuecomment-2284942961

**Comment by runxel** [comment]
I'm also on Sonoma. Still doesn't work.
Two clues:
1. It only works with documents who are in "Downloads", not if anywhere else.
2. Opening (read: double click) a Stapler doc does not open the items and closes. Instead it just shows the UI with the items that are in the doc.

**Comment by gingerbeardman** [comment]
Given that the app works, the problem is specific to your setup. We need to figure out what/why, so answers or results to the troubleshooting steps are essential :)

Please try the following approach to figuring if the app is running in translocation mode ([from here](https://eclecticlight.co/2024/05/17/how-can-you-tell-when-an-app-is-running-in-translocation/))

`ps xw | grep Stapler`

I see:
```
% ps xw | grep Stapler
59298   ??  S      0:00.36 /Applications/Stapler.app/Contents/MacOS/Stapler
```

For an app running under Translocation, you'd see something like 
```
% ps xw | grep Stapler
59298   ??  S      0:00.36 /private/var/folders/x4/ehrgwy6eg23j/T/AppTranslocation/123A-567B/d/Stapler.app/Contents/MacOS/Stapler
```

**Comment by gingerbeardman** [comment]
Good news! I've been able to set my system to reproduce this.

Will fix it as soon as possible.

**Comment by runxel** [comment]
Very cool!
What's the culprit?

Might not be relevant anymore, but this is what I got:
```terminal
~►ps xw | grep Stapler
33128   ??  S      0:00.24 /Applications/Stapler.app/Contents/MacOS/Stapler
~►ps xw | grep Stapler
33247   ??  S      0:00.18 /Users/runxel/Downloads/Stapler.app/Contents/MacOS/Stapler
```

**Comment by gingerbeardman** [comment]
please try 1.2.1 
- https://github.com/gingerbeardman/stapler/releases/tag/1.2.1

**Comment by gingerbeardman** [comment]
Woop!

Fixed in 1.2.2
https://github.com/gingerbeardman/stapler/releases/tag/1.2.2
